pkg/generator/specification_test.go

```diff
@@ -150,90 +147,37 @@ func TestSerialGenerateIAT(t *testing.T) {
 			iatDistribution: common.Equidistant,
 			shiftIAT:        false,
 			granularity:     common.MinuteGranularity,
-			expectedPoints: [][]float64{
-				{
-					// min 1
-					0,
-					12000000,
-					12000000,
-					12000000,
-					12000000,
-					12000000,
-				},
-				{
-					// min 2
-					0,
-					12000000,
-					12000000,
-					12000000,
-					12000000,
-					12000000,
-				},
-				{
-					// min 3
-					0,
-					12000000,
-					12000000,
-					12000000,
-					12000000,
-					12000000,
-				},
-				{
-					// min 4
-					0,
-					12000000,
-					12000000,
-					12000000,
-					12000000,
-					12000000,
-				},
-				{
-					// min 5
-					0,
-					12000000,
-					12000000,
-					12000000,
-					12000000,
-					12000000,
-				},
-			},
-			testDistribution: false,
-		},
-		{
-			testName:        "1min_25ipm_uniform_shift",
```

Comment:
These tests guarantee that we didn't screw up the random generation. I agree that checking for the exact match of random number is quite weird, but we can run sanity check for the result (sum, count, etc)